jetstream/cli.py

```diff
@@ -318,6 +337,13 @@ def convert(self, value, param, ctx):
     help="Kubernetes cluster certificate used for authenticating to the cluster",
 )
 
+recreate_enrollments_option = click.option(
+    "--recreate_enrollments",
+    "--recreate-enrollments",
+    help="Recreate the enrollments tables",
+    is_flag=True,
+)
+
```

Comment:
This flag is used to force re-creating the enrollment tables, e.g. used when running `rerun_config_changed`